tests/pytest-pypi/pytest_pypi/app.py

```diff
@@ -4,51 +4,56 @@
 import requests
 from flask import Flask, redirect, abort, render_template, send_file, jsonify
 
-PYPI_VENDOR_DIR = os.environ.get('PYPI_VENDOR_DIR', './pypi')
-PYPI_VENDOR_DIR = os.path.abspath(PYPI_VENDOR_DIR)
-
 app = Flask(__name__)
 session = requests.Session()
 
 packages = {}
 
 
 class Package(object):
```

Comment:
The main difference here is that `releases` is now a dict of `{"release name": "path to file"}` and a new `json` property that tries to find an `api.json` in any folder containing one of the releases.

Reply:
This will allow us to build out testing infrastructure which actually includes testing of the json api, which seems like it might be helpful....

Reply:
Yeah, I was surprised I didn't have any tests to fix for the `pytest-pypi` package itself. 😅